llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -1665,7 +1666,9 @@ class TargetTransformInfo {
   /// will contain additional information - whether the intrinsic may write
   /// or read to memory, volatility and the pointer.  Info is undefined
   /// if false is returned.
```

Comment:
We may update the comment to indicate the relationship between return value and `Interesting`?